test/osprovider.go

```diff
@@ -193,7 +193,7 @@ func (puipui *PuiPuiProvider) Fetch(destDir string) error {
 		return err
 	}
 	log.Infof("puipui initramfs: %s", puipui.initramfs)
-	puipui.kernelArgs = "console=hvc0"
+	puipui.kernelArgs = "console=quiet"
```

Comment:
Not sure if `console=quiet` can have adverse side-effects as `console=` is parsed by the kernel, and `quiet` is most likely an invalid name for a console. I used `quiet` (without `console=`) in my suggestion as this is a valid kernel option, and vfkit’s code did not like to have an empty kernel commandline. I’d guess `dummy` would also work.
I’d prefer we don’t use `console=quiet` as this could be confusing (unless this is a valid `console` option).
This could go in a separate commit before this one so that the commit log can have an explanation *why* we make that change.

Reply:
Yes, sorry, it was my mistake, fixed.